paasta_tools/cli/cmds/status.py

```diff
@@ -259,7 +261,24 @@ def report_status_for_cluster(
                 for line in status.rstrip().split('\n'):
                     paasta_print('    %s' % line)
 
-    paasta_print(report_invalid_whitelist_values(instance_whitelist, seen_instances, 'instance'))
+            paasta_print('Remote runs:')
```

Comment:
The other 3 paasta status backends (chronos/marathon/paasta_native) are implemented in paasta_serviceinit.py -- if you were to do your status call there as well then we could avoid an extra SSH call.